torntools/manifest.json

```diff
@@ -220,6 +220,22 @@
 			"js": ["scripts/content/loader/ttAttackLoader.js"],
 			"run_at": "document_end"
 		},
+		{
+			"matches": ["https://www.torn.com/loader.php?sid=view*"],
+			"js": ["scripts/content/loader/ttCasinoStatistics.js"],
+			"run_at": "document_end"
+		},
+		{
+			"matches": ["https://www.torn.com/bookies.php*"],
+			"include_globs" : ["*#/stats/*"],
+			"js": ["scripts/content/loader/ttCasinoStatistics.js"],
+			"run_at": "document_end"
+		},
+		{
+			"matches": ["https://www.torn.com/loader.php?sid=view*"],
```

Comment:
Why add it twice?

Reply:
That was actually intended for some other page, Fixed.